### DISCUSSION TITLE
报错连接不上....

### DISCUSSION THREAD

**Discussion by Yuedadsix** [v2fly/v2ray-core#1414] [Q&A] [answered]
V2Ray 4.31.0 (V2Fly, a community-driven edition of V2Ray.) Custom (go1.15.2 windows/386)
A unified platform for anti-censorship.
main: failed to read config files: [stdin:] > v2ray.com/core/main/json: failed to execute v2ctl to convert config file. > v2ray.com/core/common/platform/ctlcmd: failed to execute v2ctl: 
v2ctl> Read config:  stdin:
v2ray.com/core/infra/conf: unknown cipher method: rc4-md5 > exit status -1
有一阵没用了，再用的时候就这样了，打开先更新了订阅，然后寻思测下速，就报这个了

**Comment by kallydev** [comment, marked as answer]
It looks like you're using shadowsocks protocol, but the `rc4-md5` cipher method is not supported. Currently, v2fly-core provides shadowsocks that support only the following cipher methods.

https://github.com/v2fly/v2ray-core/blob/054e6679830885c94cc37d27ab2aa96b5b37e019/proxy/shadowsocks/config.go#L210-L223
